lib/charms/grafana_agent/v0/cos_agent.py

```diff
@@ -735,7 +716,7 @@ def metrics_jobs(self) -> List[Dict]:
     def snap_log_endpoints(self) -> List[SnapEndpoint]:
         """Fetch logging endpoints exposed by related snaps."""
         plugs = []
-        for data in self._remote_data:
+        for data, topology in self._remote_data:
```

Comment:
same as above

Reply:
with the added concern that, as far as I can tell, the `topology` variable is not being accessed

Reply:
Changed to `_`.